<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subwidget2</class>
 <widget class="QWidget" name="subwidget2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>311</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>比较结果：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>311</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>331</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输入二：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>二进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>41</y>
     <width>326</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入一：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>二进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>620</y>
     <width>295</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>完成输入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>清空输入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>401</y>
     <width>326</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>最大公约数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_9">
      <property name="text">
       <string>二进制</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QRadioButton" name="radioButton_10">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioButton_11">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_12">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>511</y>
     <width>326</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>最小公倍数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_13">
      <property name="text">
       <string>二进制</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QRadioButton" name="radioButton_14">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioButton_15">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_16">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
